go.mod

```diff
@@ -119,4 +119,4 @@ require (
 	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
-)
+)
```

Comment:
As you do not have a change in this file. I would fix the change trailing space or newline. To move this file out from list files changed.